x/leveragelp/keeper/position_close.go

```diff
@@ -52,9 +52,11 @@ func (k Keeper) ForceCloseLong(ctx sdk.Context, position types.Position, pool ty
 
 	positionOwner := sdk.MustAccAddressFromBech32(position.Address)
 
+	// TODO This means bot failed to close position on time, need to forcefully close the position
```

Comment:
so instead of throwing error here, shouldn't we close with remaining assets ?

Reply:
Yes. But have to design it more carefully. So whenever there is trigger where we update stop loss price or any other, we do all necessary checks, such as updating/payment of interests, checking if unhealthy and closing if so.

It's related to leverage lp, so didn't do in this PR